shell/platform/darwin/ios/framework/Source/VsyncWaiterIosTest.mm

```diff
@@ -0,0 +1,79 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+
+#include "flutter/fml/thread.h"
+
+#import "flutter/shell/platform/darwin/common/framework/Headers/FlutterMacros.h"
+#import "flutter/shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.h"
+
+FLUTTER_ASSERT_NOT_ARC
+namespace {
+fml::RefPtr<fml::TaskRunner> CreateNewThread(std::string name) {
+  auto thread = std::make_unique<fml::Thread>(name);
+  auto runner = thread->GetTaskRunner();
+  return runner;
+}
+}
+
+@interface VsyncWaiterIosTest : XCTestCase
+@end
+
+@implementation VsyncWaiterIosTest
+
+- (void)testCreate {
+  auto thread_task_runner = CreateNewThread("VsyncWaiterIosTest");
+  auto callback = [](std::unique_ptr<flutter::FrameTimingsRecorder> recorder) {};
+  VSyncClient* vsyncClient = [[[VSyncClient alloc] initWithTaskRunner:thread_task_runner
+                                                             callback:callback] autorelease];
+  XCTAssertNotNil(vsyncClient);
```

Comment:
I don't think this test is very valuable since your other tests validate that it's not nil. Can you instead test the case where `CADisableMinimumFrameDurationOnPhone` is not present into the plist?

Reply:
done